tests/foreman/ui/test_location.py

```diff
@@ -565,8 +577,13 @@ def test_positive_remove_subnet(self):
                         mask='255.255.255.0',
                     ).create()
                     self.assertEqual(subnet.name, subnet_name)
-                    set_context(session, org=ANY_CONTEXT['org'])
-                    make_loc(session, name=loc_name, subnets=[subnet_name])
+                    set_context(session, org=self.org_.name)
```

Comment:
@sthirugn Because it uses a different organization instead of the default, we need either to set_context or create a new user in the setup with the new org as default.

I included it in my comments below.